### PR TITLE
Update README.md (--watch and troubleshoot)

### PR DESCRIPTION
**Pull request by ppozniak**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -176,3 +177,8 @@ bots. If you are a beginner developer, please use this boilerplate.
 
 - There is a step by step walkthrough to learn how to create Discord bots with
   Discordeno on our website! https://discordeno.mod.land/stepbystep
+
+---
```
Comment (ayntee) [inline README.md:181]:
```suggestion
---

```

File: README.md [hunk 2]
```diff
@@ -176,3 +177,8 @@ bots. If you are a beginner developer, please use this boilerplate.
...
+## Troubleshooting
```
Comment (ayntee) [inline README.md:183]:
```suggestion
## Troubleshooting

```

File: README.md [hunk 3]
```diff
@@ -176,3 +177,8 @@ bots. If you are a beginner developer, please use this boilerplate.
...
+### `Module “./deps” has no exported member X deno-ts(2305)`
```
Comment (ayntee) [inline README.md:184]:
```suggestion
### `Module “./deps” has no exported member X deno-ts(2305)`

```

File: README.md [hunk 4]
```diff
@@ -67,8 +67,9 @@ export const configs = {
 };
 ```
 
-4. Start the bot
-   `deno run --allow-read --allow-write --allow-net --import-map=./import-map.json mod.ts`
+4. Start the bot using either:
+* `deno run -A --quiet mod.ts` or
```
Comment (ayntee) [inline README.md:71]:
```suggestion

* `deno run -A --quiet mod.ts` or
```

File: README.md [hunk 5]
```diff
@@ -67,8 +67,10 @@ export const configs = {
 };
 ```
 
-4. Start the bot
-   `deno run --allow-read --allow-write --allow-net --import-map=./import-map.json mod.ts`
+4. Start the bot using either:
+
+* `deno run -A --quiet mod.ts` or
```
Comment (ayntee) [inline README.md:72]:
```suggestion
    - `deno run -A --quiet mod.ts` or
```

File: README.md [hunk 6]
```diff
@@ -67,8 +67,10 @@ export const configs = {
 };
 ```
 
-4. Start the bot
-   `deno run --allow-read --allow-write --allow-net --import-map=./import-map.json mod.ts`
+4. Start the bot using either:
+
+    - `deno run -A --quiet mod.ts` or
+* `deno run -A --quiet --unstable --watch mod.ts` to enable automatic reloading on changes
```
Comment (ayntee) [inline README.md:73]:
```suggestion
    - `deno run -A --quiet --unstable --watch mod.ts` to enable automatic reloading on changes
```

File: README.md [hunk 7]
```diff
@@ -67,8 +67,9 @@ export const configs = {
 };
 ```
 
-4. Start the bot
-   `deno run --allow-read --allow-write --allow-net --import-map=./import-map.json mod.ts`
+4. Start the bot using either:
+* `deno run -A --quiet mod.ts` or
+* `deno run -A --quiet --unstable --watch mod.ts` to enable automatic reloading on changes
```
Comment (itohatweb) [inline README.md]: Is this needed? Most of the time you can use eg. `!reload commands` to load your changes. Restarting the bot on every file change can be time consuming and is not recommended (but most likely you won't hit the 1000 identify limit).

File: README.md [hunk 8]
```diff
@@ -176,3 +178,11 @@ bots. If you are a beginner developer, please use this boilerplate.
...
 - There is a step by step walkthrough to learn how to create Discord bots with
   Discordeno on our website! https://discordeno.mod.land/stepbystep
+
+---
+
+## Troubleshooting
+
+### `Module “./deps” has no exported member X deno-ts(2305)`
+
+This is most likely because you don’t have the modules cached. Run `deno cache mod.ts` and then reload your code editor.
```
Comment (ayntee) [inline README.md:188]: Can you remove this section entirely? We no longer use a `deps.ts` file...

Comment (ppozniak) [reply README.md:188]: lol

File: README.md (hunk 7 above)
Comment (ayntee) [reply README.md]: `!reload` command does not really automate the command, you are still "running" the command to reload all of the commands; on the other hand, `--watch` flag restarts the process once a file is updated and saved...

**Pull request closed without merging**